app/representers/backend_api_representer.rb

```diff
@@ -10,6 +10,7 @@ module BackendApiRepresenter
   property :system_name
   property :description
   property :private_endpoint
+  property :managed_by
```

Comment:
I was thinking that maybe would be better to expose this in the API as an annotation. Just for now limit the usage to the "managed by" use case. And only in the UI have a special handling of that annotation. If we try to handle this annotation as a special property, it appears to me there is too much non-standard stuff going on. 

I mean instead of having `property :managed_by` to just expose existing annotations. And let UI do whatever it wants with these.

WDYT?

Reply:
Yeah, that's what I tried first but I found that when I try to change the value for an existing annotation, Rails creates a new one instead of replacing the existing one. That's why I opted for accepting `managed_by` directly. But yes I think accepting annotations as parameters is better. I'll try to make up something to accept annotations properly.

Reply:
I made some changes. Now the endpoints accept the `annotations` param:

```json
 "annotations": [
  { "name": "managed", "value": "operator" }
 ]
 ```